package.json

```diff
@@ -190,6 +191,18 @@
           "inventory-storage.authority-source-files.item.delete"
         ],
         "visible": true
+      },
+      {
+        "permissionName": "ui-marc-authorities.settings.version-history",
+        "displayName": "Settings (MARC authority): Configure version history",
+        "subPermissions": [
+          "settings.marc-authorities.enabled",
+          "audit.config.groups.settings.collection.get",
+          "audit.config.groups.settings.audit.authority.collection.get",
+          "audit.config.groups.settings.item.put",
+          "audit.config.groups.settings.audit.authority.item.put"
```

Comment:
I think we don't need `audit.config.groups.settings.audit.authority.item.put` permission for now. This is only useful in cases where some users should be able to only edit some settings, and not others.
The UI permission for now is general so we can just include a general `audit.config.groups.settings.item.put` permission

Reply:
I edit the specific setting, the `audit.config.groups.settings.audit.authority.item.put` is required permission, if it is missing, an error is returned.